### DISCUSSION TITLE
Application Version

### DISCUSSION THREAD

**Discussion by zmottie** [hassio-addons/addon-unifi#458] [Ideas]
Would it at all be possible to allow the network application deb url to be a configuration value?

It would allow running of older (or newer) releases of the Unifi software without needing a new version of the add-on.

Probably an edge case but I'd like to test a few features on an old Unifi build and the Web-UI telling me there's an update available triggers me so I thought I'd ask.

thanks again!

**Comment by frenck** [comment]
> Would it at all be possible to allow the network application deb url to be a configuration value?

Nope.

> Probably an edge case but I'd like to test a few features 

This add-on is not suitable for such cases.

../Frenck
